services/core-api/app/api/now_applications/resources/now_application_export_resource.py

```diff
@@ -167,53 +216,86 @@ def get_phone(contact):
             return contact
 
         def format_currency(value):
-            return f'${float(value):,.2f}' if value and is_number(value) else value
+            return f'${float(value):,.2f}' if value != None and is_number(value) else value
 
         def format_boolean(value):
             return 'Yes' if value else 'No'
 
-        def get_reclamation_summary(now_application):
+        def get_reclamation_summary(now_application, render):
             summary = []
             activity_types = ActivityType.get_all()
-            reclamation_activity_types = (activity_type for activity_type in activity_types if activity_type.activity_type_code not in ['water_supply', 'blasting_operation'])
+            reclamation_activity_types = (
+                activity_type for activity_type in activity_types
+                if activity_type.activity_type_code not in ['water_supply', 'blasting_operation'])
             for activity_type in reclamation_activity_types:
-                if now_application.get(activity_type.activity_type_code):
-                    summary.append({
-                        'activity':
-                        activity_type.description,
-                        'total':
-                        now_application[activity_type.activity_type_code].get(
-                            'calculated_total_disturbance', EMPTY_FIELD),
-                        'cost':
-                        format_currency(now_application[activity_type.activity_type_code].get(
-                            'reclamation_cost', None))
-                    })
+                activity_type_code = activity_type.activity_type_code
+                if now_application.get(activity_type_code) and render.get(activity_type_code):
+                    activity = activity_type.description
+                    total = now_application[activity_type_code].get('calculated_total_disturbance')
+                    cost = format_currency(
+                        now_application[activity_type_code].get('reclamation_cost'))
+                    summary.append({'activity': activity, 'total': total, 'cost': cost})
             return summary
 
-        def get_applicable_now_activities(now_application):
+        def get_renderable_now_sections(now_application):
             conditional_sections = [
-                'sand_gravel_quarry_operation', 'surface_bulk_sample',
-                'cut_lines_polarization_survey', 'underground_exploration', 'placer_operation'
+                'sand_and_gravel', 'surface_bulk_sample', 'cut_lines_polarization_survey',
+                'underground_exploration', 'placer_operation'
             ]
```

Comment:
Should 'camp' be in this list?

Reply:
Oh disregard - This is just for the conditional activities. I see 'Camp' is in the "hideable" section.